WeakAuras/RegionTypes/RegionPrototype.lua

```diff
@@ -203,8 +203,8 @@ local function SoundPlay(self, options)
     return
   end
 
-  local fadeoutTime = options.sound_fade and options.sound_fade * 1000 or 0
-  self:SoundStop(fadeoutTime);
+  self.soundFadeout = options.sound_type == "Stop" and options.sound_fade and options.sound_fade * 1000 or 0
```

Comment:
Imho it was fine as a parameter to SoundStop()

Reply:
I thought it was nice if someone required it for custom aura things.
Though I missed that there already is `self.soundOptions = options`, so this change was indeed not necessary.